ChidiChuks/AutoMart
# #166329760 list of automobile products

#### What does this PR do?

- Shows list of automobile products

#### Description of Task to be completed?

- Both `Admin` and `Users` should be able to access the automobile products page

#### How should this be manually tested?

- After cloning the repo, check for a file in the `UI` folder named `products.html` and open it on a web browser preferably `Google Chrome`

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?

- [#166329760](https://www.pivotaltracker.com/story/show/166329760)

#### Screenshots (if appropriate)
#### Questions:
